lib/shared-method.js

```diff
@@ -137,6 +137,12 @@ function SharedMethod(fn, name, sc, options) {
     this.errors = [this.errors];
   }
 
+  if (/^prototype\./.test(name)) {
+    var msg = 'Incorrect API usage. Shared methods on prototypes should be ' +
+      'created via "new SharedMethod(fn, "name", { isStatic: false })';
```

Comment:
I think the quote characters are a bit off here - you use the same quote char for delimiting the code snippet and strings inside the snippet, and also the closing `"` is missing. Let's do this instead:

```
'created via `new SharedMethod(fn, "name", { isStatic: false })`';
```